server/features/processed_items_report.feature

```diff
@@ -1,158 +1,210 @@
-Feature: Processed Items Report
+Feature: Processed published items
      
      @auth
      Scenario: Processed published items
-        Given "desks"
+     	Given "roles"
+     	"""
+     	[{"name": "Admin", "privileges": {"ingest":  1, "archive": 1, "fetch": 1, "publish": 1}}]
+     	"""
+     	Given "desks"
         """
         [{"name": "Sports Desk"}]
         """
+     	Given "users"
+        """
+        [{"_id":"58a2b6b6f078e40de225913d","username": "user1", "password": "password", "email": "user1@domain.com", "sign_off": "fb", "user_type": "administrator","role": "#roles._id#"}]
```

Comment:
Is it safe here to put the user_id in the dict? I think you could  give the usernames in the dict and then retrieve them by their username: smth like this: `When we find for "users" the id as "user_admin" by "where={"username": "admin"}"`